config/routes.rb

```diff
@@ -301,9 +301,6 @@
       resources :calendar, controller: 'calendars', only: [:index]
     end
 
-    # Prevent the /*state route from overriding Rails' new
-    # view which is being used for e.g. duplicating work packages.
-    resources :work_packages, only: [:new]
 
     resources :work_packages, only: [:create] do
```

Comment:
There are other references to no longer existing actions in the routes, e.g.:
https://github.com/oliverguenther/openproject/blob/features/16364-remove_rails_views/config/routes.rb#L428